From ec3d28d82e4120ac3dfc943e53c613caab74ea54 Mon Sep 17 00:00:00 2001
From: crazy <crazy@frugalware.org>
Date: Mon, 21 May 2018 23:20:12 +0200
Subject: [PATCH] Revert "mkfs: enable sparse inodes by default" This reverts
 commit 9cf846b514ff335470f130d635eff525386bab21.

-- that breaks xfs as roofs .. grub does not support these yet
---
 mkfs/xfs_mkfs.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mkfs/xfs_mkfs.c b/mkfs/xfs_mkfs.c
index 78d0ce5d..1ca6a2d1 100644
--- a/mkfs/xfs_mkfs.c
+++ b/mkfs/xfs_mkfs.c
@@ -1996,7 +1996,7 @@ _("finobt not supported without CRC support\n"));
 		}
 		cli->sb_feat.finobt = false;
 
-		if (cli->sb_feat.spinodes && cli_opt_set(&iopts, I_SPINODES)) {
+		if (cli->sb_feat.spinodes) {
 			fprintf(stderr,
 _("sparse inodes not supported without CRC support\n"));
 			usage();
@@ -3811,7 +3811,7 @@ main(
 			.crcs_enabled = true,
 			.dirftype = true,
 			.finobt = true,
-			.spinodes = true,
+			.spinodes = false,
 			.rmapbt = false,
 			.reflink = false,
 			.parent_pointers = false,
-- 
2.17.0

